src/app/components_ngrx/planner-query/planner-query.component.html

```diff
@@ -15,18 +15,37 @@ <h4>Please wait, we are loading your data.</h4>
 	</work-item-preview-panel>
 
 	<div #querySearch class="f8-query__input-wrapper">
-		<span
-			*ngIf="searchQuery !== '' && !query.disabled"
-			class="pficon-close close f8-query__clear-button"
-			tooltip="Press ctrl+backspace to clear"
-			placement="left"
-			(click)="searchQuery='';"
-			></span>
-		<input #query class="f8-query__input"
-			[disabled]="disableInput"
-			(keydown)="fetchWorkItemForQuery($event, query.value)"
-			[(ngModel)]="searchQuery"
-				placeholder="Enter your Query...">
+		<div class="dropdown"
+			[isOpen]="isSuggestionDropdownOpen"
+			[autoClose]="false"
+			dropdown>
+			<span
+				*ngIf="searchQuery !== '' && !queryInput.disabled"
+				class="close f8-query__clear-button"
+				[class.pficon-spinner2]="valueLoading"
```

Comment:
I'm not sure this adds much value. It continuously blinks while typing. Considering the suggestions is an added value that appears automatically, i don't think we need to show a spinner while it's loading?
We can keep it, I'd have UXD comment on this behavior.

Reply:
We added the spinner here because for query `assignee: USER_NAME` we make a backend request to fetch the users and it takes some time.

Reply:
But it is happening on every keystroke which is a bit excessive. IIRC i inspected the network and saw no requests being made while the spinner was blinking as I typed.

Reply:
I can't reproduce this one. In my Case, request is being made to backend